libs/dh/admin/feature-permissions/src/lib/details/tabs/dh-admin-permission-roles.component.html

```diff
@@ -14,68 +14,62 @@
 See the License for the specific language governing permissions and
 limitations under the License.
 -->
-<ng-container *rxLet="loading as isLoading">
-  <watt-card
-    variant="solid"
-    *transloco="let t; read: 'admin.userManagement.permissionDetail.tabs.userRoles'"
-  >
-    @if (isLoading) {
-      <div class="spinner">
-        <watt-spinner />
-      </div>
-    }
-
-    @if (!isLoading) {
-      <watt-card-title *rxLet="userRoles.length as userRoleCount">
-        @if (userRoleCount === 1 && !error) {
-          <h4>
-            {{ t("rolesSingular", { userRoleCount }) }}
-          </h4>
-        } @else if (userRoleCount !== 1 && !error) {
-          <h4>
-            {{ t("rolesPlural", { userRoleCount }) }}
-          </h4>
-        }
+<watt-card
+  variant="solid"
+  *transloco="let t; read: 'admin.userManagement.permissionDetail.tabs.userRoles'"
+>
+  @if (loading()) {
+    <div class="spinner">
+      <watt-spinner />
+    </div>
```

Comment:
Nitpick: @let ?

Reply:
I considered it too bug didn't see a big difference since it will only save us a couple of characters - (). In terms of perf, calling the same Signal multiple times shouldn't be an issue.